# GOAL
float64 side_length # Longitud del lado del tri치ngulo
---
# RESULT
bool success # True si el tri치ngulo se ha ejecutado correctamente
---
# FEEDBACK
string current_movement # Movimiento actual: lado, giro...
int64 movement_id # Id del movimiento actual: lado 1, giro 1...
float64 movement_state # Posici칩n u orientaci칩n actual
